java-client/build.gradle.kts

```diff
@@ -30,8 +30,8 @@ plugins {
 }
 
 java {
-    targetCompatibility = JavaVersion.VERSION_1_8
-    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_10
```

Comment:
Why 1.8 was dropped? The minimum should be JDK-8 for 1.x

Reply:
Reverted back to 1.8